### PR TITLE
Improved lambda getting started docs

### PR DESCRIPTION
**Pull request by AlexanderWert**
Depends on: 
- https://github.com/elastic/apm-aws-lambda/pull/158

Related:
- https://github.com/elastic/apm-agent-java/pull/2556

### CODE REVIEW

File: docs/lambda/configure-lambda-widget.asciidoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
+++++
+<div class="tabs" data-tab-group="os">
+  <div role="tablist" aria-label="dependency">
+    <button role="tab"
+            aria-selected="true"
+            aria-controls="console-tab-lambda-java-config"
```
Comment (felixbarny) [inline docs/lambda/configure-lambda-widget.asciidoc:6]: There are some mentions of "Java" here.

**Review by basepi — DISMISSED** [review]
I would really like to avoid adding binary files (the .png) to the repo. Does it make sense to move them to a central repo? Perhaps something like `elastic/apm-artifacts`, and then pull them into the docs from there?

134KB isn't a lot, but every binary file will be downloaded in every clone from now to the end of time, and if the image changes git can't diff them so now both versions will be included in the repo forever.

**Comment by bmorelli25** [comment]
> > I would really like to avoid adding binary files (the .png) to the repo. Does it make sense to move them to a central repo? Perhaps something like elastic/apm-artifacts, and then pull them into the docs from there?

> If there's another way to include the image I'm up for it. @bmorelli25 do you have an idea what we could do instead of uploading the image in this repo and embedding it in the docs?

We're already including a lot of content in these docs from `elastic/aws-apm-lambda/docs`. I don't see why we can't store the images there too. I'll add them to https://github.com/elastic/apm-aws-lambda/pull/158, cherry-pick to 8.1, and rerun the build.

File: docs/serverless.asciidoc [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+==== Prerequisites
+
+You need an APM server where the APM data will be sent to. {apm-guide-ref}/apm-quick-start.html[Set up an APM Server] if you have not not done so yet. For the best-possible performance we recommend to setup APM on Elastic Cloud in the _same AWS region_ as your AWS Lambda functions. 
+
+[float]
+==== Step 1: Setup the APM Lambda Extension
+
+include::{apm-aws-lambda-root}/docs/lambda-selector/lambda-attributes-selector.asciidoc[]
+
+Add the {apm-guide-ref}/aws-lambda-arch.html[APM Lambda Extension] as an https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html[AWS Lambda Layer] to your AWS Lambda function.
```
[the remark this answers is not in the document]
Comment (bmorelli25) [reply docs/serverless.asciidoc:19]: 👍 I'll fix this when I move the image.

[the remark this answers is not in the document]
Comment (bmorelli25) [reply docs/serverless.asciidoc:19]: Fixed in 2629fdc and 2e4f0c6.

**Comment by basepi** [comment]
Honestly it's fine if we want to store them here. I just have worked on projects with very large clone times due to binary files and want to at least discuss alternatives before we merge them.

**Review by basepi — APPROVED** [review]
These docs are excellent! Love the tabbed sections for the various ways lambda can be configured!

**Pull request merged**